Fenrir/Tools.fs

```diff
@@ -41,3 +41,12 @@ let stringToByte (s: String): byte[] =
             Byte.Parse(s.AsSpan(charIndex, 2), NumberStyles.AllowHexSpecifier, provider = CultureInfo.InvariantCulture)
         )
     | n -> failwithf "String of invalid length %d: %s" n s
+
+type BinaryReader with
+    member reader.GetBigEndian() : int =
```

Comment:
_(Minor)_ Let's rename this to `ReadBigEndianInt`? Otherwise, it's confusing (I genuinely was confused by this name while reviewing this).